var/spack/repos/builtin/packages/rocblas/package.py

```diff
@@ -75,11 +81,16 @@ def cmake_args(self):
             '-DTensile_COMPILER=hipcc',
             '-DTensile_ARCHITECTURE={0}'.format(archs),
             '-DTensile_LOGIC=asm_full',
-            '-DTensile_CODE_OBJECT_VERSION=V3',
-            '-DBUILD_WITH_TENSILE_HOST=OFF'
+            '-DTensile_CODE_OBJECT_VERSION=V3'
         ]
 
-        if '@3.7.0' in self.spec:
+        if '@3.7.0:' in self.spec:
             args.append('-DTensile_LIBRARY_FORMAT=msgpack')
 
+        # Potentially this already works in 3.7.0, but was not tested
+        if '@3.8.0:' in self.spec:
+            args.append('-DBUILD_WITH_TENSILE_HOST=ON')
+        else:
```

Comment:
this is recommendation from the rocblas internal team -
ROCm 3.7 can use -DBUILD_WITH_TENSILE_HOST=ON while ROCm 3.8 and later must use -DBUILD_WITH_TENSILE_HOST=ON .  ROCm 3.5 must use -DBUILD_WITH_TENSILE_HOST=OFF .  Thanks.

Reply:
i have tested the recipe for 3.8.0 and it works .

Reply:
need fix for flake8 error

Reply:
Thanks, it's done